diff -uNr libuv-1.30.1/CMakeLists.txt libuv-1.30.1_patched/CMakeLists.txt
--- libuv-1.30.1/CMakeLists.txt	2019-07-02 16:55:03.000000000 +0200
+++ libuv-1.30.1_patched/CMakeLists.txt	2019-08-06 09:43:12.443224127 +0200
@@ -8,6 +8,7 @@
   list(APPEND uv_cflags -fvisibility=hidden --std=gnu89)
   list(APPEND uv_cflags -Wall -Wextra -Wstrict-prototypes)
   list(APPEND uv_cflags -Wno-unused-parameter)
+  list(APPEND uv_cflags -fPIC)
 endif()
 
 set(uv_sources
