diff -uNr libuv-1.46.0/CMakeLists.txt libuv-1.46.0_patched/CMakeLists.txt
--- libuv-1.46.0/CMakeLists.txt	2023-07-18 15:53:13.014297763 +0200
+++ libuv-1.46.0_patched/CMakeLists.txt	2023-07-18 15:55:56.214968383 +0200
@@ -721,26 +721,26 @@
   endif()
 endif()
 
-# Now for some gibbering horrors from beyond the stars...
-foreach(lib IN LISTS uv_libraries)
-  list(APPEND LIBS "-l${lib}")
-endforeach()
-string(REPLACE ";" " " LIBS "${LIBS}")
-# Consider setting project version via project() call?
-file(STRINGS configure.ac configure_ac REGEX ^AC_INIT)
-string(REGEX MATCH "([0-9]+)[.][0-9]+[.][0-9]+" PACKAGE_VERSION "${configure_ac}")
-set(UV_VERSION_MAJOR "${CMAKE_MATCH_1}")
-
-set(includedir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR})
-set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
-set(prefix ${CMAKE_INSTALL_PREFIX})
-configure_file(libuv-static.pc.in libuv-static.pc @ONLY)
+## Now for some gibbering horrors from beyond the stars...
+#foreach(lib IN LISTS uv_libraries)
+#  list(APPEND LIBS "-l${lib}")
+#endforeach()
+#string(REPLACE ";" " " LIBS "${LIBS}")
+## Consider setting project version via project() call?
+#file(STRINGS configure.ac configure_ac REGEX ^AC_INIT)
+#string(REGEX MATCH "([0-9]+)[.][0-9]+[.][0-9]+" PACKAGE_VERSION "${configure_ac}")
+#set(UV_VERSION_MAJOR "${CMAKE_MATCH_1}")
+#
+#set(includedir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR})
+#set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
+#set(prefix ${CMAKE_INSTALL_PREFIX})
+#configure_file(libuv-static.pc.in libuv-static.pc @ONLY)
 
 install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
-install(FILES LICENSE-extra DESTINATION ${CMAKE_INSTALL_DOCDIR})
-install(FILES ${PROJECT_BINARY_DIR}/libuv-static.pc
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+#install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
+#install(FILES LICENSE-extra DESTINATION ${CMAKE_INSTALL_DOCDIR})
+#install(FILES ${PROJECT_BINARY_DIR}/libuv-static.pc
+#        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(TARGETS uv_a EXPORT libuvConfig
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(EXPORT libuvConfig
