diff -uNr libuv-1.27.0/CMakeLists.txt libuv-1.27.0-patched/CMakeLists.txt
--- libuv-1.27.0/CMakeLists.txt	2019-04-03 13:31:09.577269414 +0200
+++ libuv-1.27.0-patched/CMakeLists.txt	2019-04-03 13:32:35.487086516 +0200
@@ -371,23 +371,23 @@
            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-if(UNIX)
-  # Now for some gibbering horrors from beyond the stars...
-  include(GNUInstallDirs)
-  foreach(x ${uv_libraries})
-    set(LIBS "${LIBS} -l${x}")
-  endforeach(x)
-  file(STRINGS configure.ac configure_ac REGEX ^AC_INIT)
-  string(REGEX MATCH [0-9]+[.][0-9]+[.][0-9]+ PACKAGE_VERSION "${configure_ac}")
-  set(includedir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR})
-  set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
-  set(prefix ${CMAKE_INSTALL_PREFIX})
-  configure_file(libuv.pc.in ${CMAKE_CURRENT_BINARY_DIR}/libuv.pc @ONLY)
+#if(UNIX)
+#  # Now for some gibbering horrors from beyond the stars...
+#  include(GNUInstallDirs)
+#  foreach(x ${uv_libraries})
+#    set(LIBS "${LIBS} -l${x}")
+#  endforeach(x)
+#  file(STRINGS configure.ac configure_ac REGEX ^AC_INIT)
+#  string(REGEX MATCH [0-9]+[.][0-9]+[.][0-9]+ PACKAGE_VERSION "${configure_ac}")
+#  set(includedir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR})
+#  set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
+#  set(prefix ${CMAKE_INSTALL_PREFIX})
+#  configure_file(libuv.pc.in ${CMAKE_CURRENT_BINARY_DIR}/libuv.pc @ONLY)
 
-  install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-  install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libuv.pc
-          DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
-  install(TARGETS uv LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
-  install(TARGETS uv_a ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-endif()
+  install(DIRECTORY include/ DESTINATION "include")
+#  install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
+#  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libuv.pc
+#          DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+#  install(TARGETS uv LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+  install(TARGETS uv_a ARCHIVE DESTINATION "lib")
+#endif()
